### PR TITLE
Rename csvSteamAchievement to toCsvString, return String

### LINKED ISSUE #11: Rename csvSteamAchievement to toCsvString, return String

### PR DESCRIPTION
**Pull request by Bladerama**
Rename csvSteamAchievement to toCsvString, return String

### CODE REVIEW

File: src/main/java/com/axlsreborn/CsvWriter.java [hunk 1]
```diff
@@ -8,35 +8,23 @@
...
     }
 
     public static void writeFile(List<SteamAchievement> steamAchievementsList, String csvFilePath, char csvDelimiter)
             throws FileNotFoundException {
         PrintWriter printWriter = new PrintWriter(csvFilePath);
 
         for (SteamAchievement steamAchievement : steamAchievementsList) {
-            java.lang.String steamAchievementStr =
-                    CsvWriter.csvSteamAchievement(steamAchievement, csvDelimiter).toString();
+            java.lang.String steamAchievementStr = CsvWriter.toCsvString(steamAchievement, csvDelimiter).toString();
```
Comment (fbis251) [inline src/main/java/com/axlsreborn/CsvWriter.java:27]:
```suggestion
           String steamAchievementStr = CsvWriter.toCsvString(steamAchievement, csvDelimiter);
```

**Comment by fbis251** [comment]
You should make `Rename csvSteamAchievement to toCsvString, return String` the ttle and `Fixes #11` the comment. I couldn't tell what 'Fixes #11' meant when I got the email notification for this PR

File: src/main/java/com/axlsreborn/CsvWriter.java [hunk 2]
```diff
@@ -8,35 +8,36 @@
...
-    public static StringBuffer csvSteamAchievement(SteamAchievement steamAchievement, char csvDelimiter) {
+    public static String toCsvString(SteamAchievement steamAchievement, char csvDelimiter) {
         StringBuffer stringBuffer = new StringBuffer();
 
-        return stringBuffer
+        stringBuffer
                 .append("\"")
                 .append(steamAchievement.getAchievementName())
-                .append("\"")
+                .append("\" ")
```
Comment (fbis251) [inline src/main/java/com/axlsreborn/CsvWriter.java:17]: How come you added the spaces here? The expected output should be
"one","two"

File: src/main/java/com/axlsreborn/CsvWriter.java [hunk 3]
```diff
@@ -8,35 +8,36 @@
...
+                .append(" \"")
                 .append(steamAchievement.getAchievementDescription())
-                .append("\"")
+                .append("\" ")
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementIconUrl())
                 .append("\"")
                 .append("\n");
```
Comment (CaffeinatedTom) [inline src/main/java/com/axlsreborn/CsvWriter.java:30]: It's been a while since I've used a string buffer, but couldn't this be 

```suggestion
                .append("\"\n");
```

File: src/main/java/com/axlsreborn/CsvWriter.java [hunk 4]
```diff
@@ -8,35 +8,36 @@
...
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementPercent())
-                .append("\"")
+                .append("\" ")
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementDescription())
```
Comment (CaffeinatedTom) [inline src/main/java/com/axlsreborn/CsvWriter.java:24]: This can be for another PR, but https://github.com/fb-help/SteamAchievements/issues/9 still seems relevant unless I'm missing where you escape commas and quotes contained within AchievementDescription.
